tests/unit/test__main.py

```diff
@@ -207,3 +327,194 @@ def test_config_runner_4(mock_args, mock_build, mock_run, caplog):
     )
     assert mock_run.call_count == 4
     assert e_info.value.args[0] == "Some of the validations raised an exception"
+
+
+@mock.patch(
+    "argparse.ArgumentParser.parse_args",
+    return_value=argparse.Namespace(**VALIDATE_COLUMN_CONFIG),
+)
+@mock.patch(
+    "data_validation.__main__.build_config_managers_from_args",
+    return_value=[
+        config_manager.ConfigManager(
+            VALIDATE_COLUMN_CONFIG, MockIbisClient(), MockIbisClient(), verbose=False
+        )
+    ],
+)
+@mock.patch("data_validation.__main__.run_validation")
+def test_successful_column_validation_with_mocked_run_validation(
+    mock_args, mock_build, mock_run
```

Comment:
Either these arguments in are in the wrong order of the `mock.patch` calls are in the wrong order. I don't suppose it matters in practice as long as they do what they do. But I guess the names should match up.

Reply:
@nj1973 
I dont see whats wrong, can you suggest what order the `mock_args, mock_build, mock_run` should be in. Im happy to make the change throughout the script

Reply:
@nj1973 Thanks for explaining, I wasn't aware of this detail.

Ill make a fix now 👍

Reply:
@nj1973 Fixed here -> https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/1329/commits/796783564a28997b77cd0f4d689c73892139314a